src/protocol/requests/addOffsetsToTxn/v1/response.js

```diff
@@ -0,0 +1,24 @@
+const { parse, decode: decodeV0 } = require('../v0/response')
+
+/**
+ * Starting in version 1, on quota violation, brokers send out responses before throttling.
+ * @see https://cwiki.apache.org/confluence/display/KAFKA/KIP-219+-+Improve+quota+communication
+ *
+ * AddOffsetsToTxn Response (Version: 1) => throttle_time_ms error_code
+ *   throttle_time_ms => INT32
+ *   error_code => INT16
+ */
+const decode = async rawData => {
+  const decoded = await decodeV0(rawData)
+
+  return {
+    ...decoded,
+    throttleTime: 0,
```

Comment:
Why is it that you're overriding `throttleTime` here?

I can see the logic that you want to say that the request wasn't throttled, and that we are supposed to throttle on our side, but it's inconsistent with what we're doing everywhere else, where we return the original value for `throttleTime` and simply copy it to `clientSideThrottleTime`.

Reply:
That said, I think I like your decision. I just wanna make sure we're consistent.

Reply:
I think I actually changed all places to set throttleTime to 0 when returning a client-side value. :)

The thinking is that "throttleTime" greater than 0 means the broker throttled for that amount; while "clientSideThrottleTime" greater 0 indicates that the client did/should do so.